saleor/settings.py

```diff
@@ -697,3 +697,6 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
 
 assert hasattr(schema_printer, "_print_object")
 schema_printer._print_object = patched_print_object
+
+# Lock time for request password reset mutation per user (seconds)
+RESET_PASSWORD_LOCK_TIME = int(os.environ.get("RESET_PASSWORD_LOCK_TIME", 15 * 60))
```

Comment:
You can do it more friendly to read:
```suggestion
RESET_PASSWORD_LOCK_TIME = int(os.environ.get("RESET_PASSWORD_LOCK_TIME", parse("15 minutes")))
```